frontend/src/components/pages/CampCreation/index.tsx

```diff
@@ -11,13 +10,29 @@ import {
   CreateFormQuestion,
   CreateCampSession,
 } from "../../../types/CampsTypes";
+import CampCreationDetails from "./CampDetails";
 
 const CampCreationPage = (): React.ReactElement => {
   /* eslint-disable */
   // All response state from the three page components.
-  const [campDetailsDummyOne, setCampDetailsDummyOne] = useState(false);
-  const [campDetailsDummyTwo, setCampDetailsDummyTwo] = useState(false);
-  const [campDetailsDummyThree, setCampDetailsDummyThree] = useState("");
+  const [campName, setCampName] = useState<string>();
```

Comment:
When the input changes, we currently get the following error:
![image](https://user-images.githubusercontent.com/57159970/205415654-1471de10-b1ce-4f07-bf4a-0cbcc6de9897.png)

Could you check if all the setters and states are correct? You might also want to use default values instead of undefined here (eg: empty strings, 0 for ints etc.)